packages/utils/ast-utils.test.js

```diff
@@ -34,28 +34,28 @@ describe("utils", () => {
 	describe("findPluginsByName", () => {
 		it("should find plugins in AST", () => {
 			const ast = j(`
-{ foo: new webpack.optimize.UglifyJsPlugin() }
+{ foo: new webpack.optimize.TerserPlugin() }
```

Comment:
Is terser plugin inside optimize? Genuine question, I can't remember

Reply:
hmm i think its in separate package e.g. `const TerserPlugin = require('terser-webpack-plugin');`